src/content/en/showcase/2017/within.md

```diff
@@ -0,0 +1,386 @@
+staying performant. Unfortunately, it wasn't as easy to implement so we'll go
+through the steps in hopes to alleviate fellow developers working in WebVR.
+
+[mattdesl.svbtle.com/material-design-on-the-gpu](https://mattdesl.svbtle.com/material-design-on-the-gpu)
+
+[mattdesl.github.io/physical-text/index.html](http://mattdesl.github.io/physical-text/index.html)
+
+[github.com/mattdesl/physical-text](https://github.com/mattdesl/physical-text)
+
+
+## 1. Generate bitmap font (.png + .fnt)
+
+<figure class="attempt-right">
+  <img src="/web/showcase/2017/images/within/3.png">
+  <figcaption>Hiero Interface</figcaption>
+</figure>
+
+[libgdx.badlogicgames.com/tools.html](https://libgdx.badlogicgames.com/tools.html)
+
+[github.com/libgdx/libgdx/wiki/Hiero](https://github.com/libgdx/libgdx/wiki/Hiero)
+
+[github.com/libgdx/libgdx/wiki/Distance-field-fonts](https://github.com/libgdx/libgdx/wiki/Distance-field-fonts)
+
+<figure class="attempt-right">
+  <div>
+    <img src="/web/showcase/2017/images/within/4.png"><img src="/web/showcase/2017/images/within/5.png">
+  </div>
+  <figcaption>Hiero Output (Bitmap PNG and .fnt file)</figcaption>
+</figure>
+
+Hiero is a bitmap font packing tool that runs with Java. The Hiero documentation
+doesn't really explain how to run it without going through a complicated build
+process. First, install Java if you haven't already. Then if double clicking on
+the runnable-hiero.jar doesn't open Hiero, try running it with this command in
+the console:
+
+    java -jar runnable-hiero.jar
+
+Once Hiero is running, open a .ttf or .otf desktop font, enter any extra
+characters you want included, change Rendering to Java to enable Effects,
+increase the Size so that your characters fill up the entire Glyph cache square,
+add a Distance Field Effect, adjust the Distance field's scale and spread. The
+scale value is like a resolution. The higher it is, the less blurry it will be,
+but the longer it will take for Hiero to render the preview. Then save your
+bitmap font. It generates a bitmap font is consisting of a .png image and an
+AngelCode .fnt font description file.
+
+## 2. Convert AngelCode to JSON
+
+Now that the bitmap font has been generated, we have to load it into our
+javascript app with Matt DesLauriers' load-bmfont npm package:
+
+[github.com/Jam3/load-bmfont](https://github.com/Jam3/load-bmfont)
+
+[npmjs.com/package/load-bmfont](https://www.npmjs.com/package/load-bmfont)
+
+We could browserify load-bmfont and use that on the front end, but instead we're
+going to run
+[load-bmfont.js](https://vr.with.in/archive/text-sdf-bitmap/load-bmfont.js) with
+Node to convert and save Hiero's AngelCode .fnt to a .json file:
+
+    npm install
+    node load-bmfont.js
+
+[roboto-bold.json](https://vr.with.in/archive/text-sdf-bitmap/fonts/roboto/bitmap/roboto-bold.json)
+
+<figure class="attempt-right">
+  <img src="/web/showcase/2017/images/within/6.png">
+  <figcaption>Example of output JSON</figcaption>
+</figure>
+
```

Comment:
add a `<div class="clearfix"></div>` here to break the funky spacing that is caused by the floating images